packages/console/src/common/ConsoleUtils.ts

```diff
@@ -32,38 +52,39 @@ class ConsoleUtils {
     return `${hours}:${minutes}:${seconds}.${milliseconds}`;
   }
 
-  static defaultHost() {
+  static defaultHost(): string {
     let defaultHost = null;
-    try {
-      const url = new URL(process.env.REACT_APP_CORE_API_URL);
+    const apiUrl = process.env.REACT_APP_CORE_API_URL;
+    if (apiUrl != null) {
+      const url = new URL(apiUrl);
```

Comment:
This is a change in behaviour - I think we still want to wrap a try/catch around the URL creation, since it can throw, eg. `new URL('__mocks__')` (which it will get with unit tests).

I'd write it something like:

```
let defaultHost = window.location.hostname;
if (apiUrl != null) {
  try {
      const url = new URL(apiUrl);
      apiUrl = url.hostname;
  } catch (error) { }

```